packages/ember-glimmer/tests/integration/syntax/if-unless-test.js

```diff
@@ -142,4 +405,79 @@ moduleFor('Syntax test: {{#unless}}', class extends RenderingTest {
     this.assertText('T1F2');
   }
 
-});
+}
+
+moduleFor('Syntax test: {{#if}}', class extends SharedConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{#if ${cond}}}${truthy}{{else}}${falsy}{{/if}}`;
+  }
+
+  ['@test it renders and hides the given block based on the conditional']() {
```

Comment:
This is the only test for without inverse (primarily because I haven't figured out how to abstract it yet)